docs/evm/using.mdx

```diff
@@ -40,7 +40,9 @@ Manually method: Add EVM on Flow as a custom network to MetaMask:
 
 6. Tap the Save button to save Flow Previewnet as a network.
 
-You should now be able to connect to the EVM on Flow by selecting it from the network selection dropdown menu
+You should now be able to connect to the EVM on Flow by selecting it from the network selection dropdown menu.
+
+Use the [Flow Previewnet Faucet](https://previewnet-faucet.onflow.org/fund-account) to create and fund a new account for testing.
```

Comment:
Can drop create since it's a Cadence feature
```suggestion
Use the [Flow Previewnet Faucet](https://previewnet-faucet.onflow.org/fund-account) to fund a new account for testing.
```